src/python/grpcio_status/grpc_status/rpc_status.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Reference implementation for status mapping in gRPC Python."""
+
+import grpc
+
+# Have to import google.protobuf before google.rpc, otherwise Python will fail
+#   to locate module rpc under google.
+import google.protobuf  # pylint: disable=unused-import
+from google.rpc import status_pb2
+
+_CODE_TO_GRPC_CODE_MAPPING = dict([(x.value[0], x) for x in grpc.StatusCode])
+_GRPC_CODE_TO_CODE_MAPPING = dict([(x, x.value[0]) for x in grpc.StatusCode])
```

Comment:
Can you just use `grpc.StatusCode[code].value[0]` instead of making this mapping? I would prefer something similar instead of `_CODE_TO_GRPC_CODE_MAPPING` as well, but don't immediately see that we have this available (would be very nice if we could do `grpc.StatusCode(0)` instead of `grpc.StatusCode((0, 'ok'))`).

Reply:
I'm not sure if we should add another public API to support code translation. So conservatively, I removed latter mapping constant, but keep the former one.